core/modules/compare/store/plugin.ts

```diff
@@ -6,8 +6,7 @@ const watchedMutations = [types.COMPARE_ADD_ITEM, types.COMPARE_DEL_ITEM, types.
 const cacheStorage = StorageManager.get('compare')
 
 const cachePersistPlugin = (mutation, state) => {
-  const mutations = watchedMutations.map(m => `compare/${m}`)
-
+  const mutations = watchedMutations
   if (mutations.includes(mutation.type)) {
```

Comment:
wouldn't it be better to remove `const mutations` and have just `watchedMutations.includes` instead?

Reply:
Yeah, totally right, we are not using this variable anyway anymore